website/docusaurus.config.js

```diff
@@ -116,8 +119,15 @@ const config = {
           { to: "/blog", label: "Blog", position: "left" },
           {
             href: "https://github.com/functionless/functionless",
-            label: "GitHub",
             position: "right",
+            className: "header-github-link",
+            "aria-label": "GitHub Repository",
+          },
+          {
+            href: "https://functionless.slack.com",
```

Comment:
Is this an invite link?

Reply:
No. Just signin/signup. I think we need to turn off the email requirement. I don't like the idea of link that will stop working after x joins.

Reply:
Agreed

Reply:
By email requirement, do you mean Functionless.org

Reply:
Yeah. Looks like we cannot open up the allow list. Only invite or Allowlist options.